package.json

```diff
@@ -4,6 +4,6 @@
   "description": "package.json intended for in-repo use only, package.json used for publishing is located in src/AutoRest.CSharp/package.json",
   "devDependencies": {
     "@microsoft.azure/autorest.testserver": "3.0.21",
-    "autorest": "3.2.1"
+    "autorest": "3.2.3"
```

Comment:
is it necessary to update autorest version for this?

Reply:
Remove change